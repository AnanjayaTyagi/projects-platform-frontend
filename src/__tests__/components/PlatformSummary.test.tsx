import * as React from "react";
import renderer from "react-test-renderer";

import PlatformSummary from "../../components/PlatformSummary";

it("renders PlatformSummary correctly", () => {
  const component = renderer.create(<PlatformSummary />).toJSON();
  expect(component).toMatchSnapshot();
});
