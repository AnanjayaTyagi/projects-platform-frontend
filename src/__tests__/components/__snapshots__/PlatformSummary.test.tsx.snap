// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders PlatformSummary correctly 1`] = `
<section
  className="project-summary-section"
>
  <h2>
    The Title Will Explain The Platform, Briefly
  </h2>
  <p>
    The intro paragraph will go into a bit more details. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Quisque a mattis purus, vel cursus justo. Ut nec pretium ex. Nullam ac ultricies orci. Nulla facilisi.
  </p>
</section>
`;
